src/assets/dataset/testData.js

```diff
@@ -0,0 +1,5976 @@
+export const testData = [

```

Comment:
Using TS with the proper defined type for testData would make the user be clear on what the data is about.